src/main/java/seedu/duke/logic/commands/circuit/TutorialCircuitCommand.java

```diff
@@ -8,6 +8,16 @@
 import java.util.logging.Level;
 
 public class TutorialCircuitCommand extends CircuitCommand implements TutorialCommand {
+    private static final String COMMAND_SUMMARY = "Here's a summary of available commands: \n"
+            + "+-----------+----------------------------+--------------------+\n"
+            + "|  Action   |           Format           |      Examples      |\n"
+            + "+-----------+----------------------------+--------------------+\n"
+            + "| Tutorial  | tutorial MODE              | tutorial boolean   |\n"
+            + "| Template  | template TEMPLATE          | template rc        |\n"
+            + "| Set       | set COMPONENT VALUE        | set r 500          |\n"
+            + "| Add       | add CONFIG COMPONENT VALUE | add parallel c 500 |\n"
+            + "| Calculate | calc EFF_VALUE             | calc ceff          |\n"
+            + "+-----------+----------------------------+--------------------+\n";
```

Comment:
Err I'm not sure but why is this summary table in TutorialCircuitCommand?
Can use the summary table from SummaryCommand, perhaps create an object of SummaryCommand and print it?